cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -438,7 +441,8 @@ def test_cluster_section_from_231_cfn(mocker, cfn_params_dict, expected_section_
                     "initial_queue_size": 0,
                     "max_queue_size": 3,
                     "placement": "cluster",
-                    "base_os": "ubuntu1404",  # NOTE: We create the config with the old base_os (no longer supported)
+                    "base_os": "ubuntu1404",
```

Comment:
I'd keep the comment. It was to highlight we have an unsupported os in the cfn params